bench/README.md

```diff
@@ -0,0 +1,16 @@
+# Benchmarks
+
+## Running
+
+* Install node modules required for the benchmark you wish to run
+* Use `npm run bench -- --help` for options
+
+For example, let's say you like to know how the performance of `toString`
+compares to what it was at version 2.x:
+
+```sh
+cd bench
+npm i sanctuary-type-classes@2.x
+cd ..
+npm run bench -- --benchmark old-vs-new --match methods.toString.*
```

Comment:
In case you are curious:

![latest-screenshot](https://user-images.githubusercontent.com/1217745/36073665-91132cb6-0f34-11e8-944c-230dfa47acc7.png)

Reply:
Formatting requests:

  - change `sh` to `console`, and prefix each command with a `$` prompt;
  - use `install` rather than `i`; and
  - use `@2.x.x` rather than `@2.x`.